website/versioned_docs/version-0.23/asyncstorage.md

```diff
@@ -4,11 +4,11 @@ title: AsyncStorage
 original_id: asyncstorage
 ---
 
-AsyncStorage is a simple, asynchronous, persistent, key-value storage system that is global to the app. It should be used instead of LocalStorage.
+AsyncStorage is a asynchronous, persistent, key-value storage system that is global to the app. It should be used instead of LocalStorage.
```

Comment:
```suggestion
AsyncStorage is an asynchronous, persistent, key-value storage system that is global to the app. It should be used instead of LocalStorage.
```